app/index.html

```diff
@@ -29,11 +29,14 @@
       }
 
       .video-container.visible {
+        height: 100vh;
```

Comment:
Could you drop this bit? As long as we have the max height specified, we don't need an explicit height, specifying one breaks the layout a bit. Looks good without it though :smiley: